### PR TITLE
Admonition Customization tutorial

### PR DESCRIPTION
**Pull request by dillp1**
This PR is set to add an admonition customization tutorial. The tutorial is aimed at beginners who have little or no prior CSS or HTML experience. The tutorial guides the user through the HTML using Inspect, and teaches them how to edit the simple CSS of Docusaurus to edit any html element.

Constructive feedback is appreciated for the clarity and correctness of the tutorial. I would be more than happy to add or change any content!

### CODE REVIEW

**Review by bohyunjung — CHANGES_REQUESTED** [review]
Nice work @dillp1!

If you find my review helpful, please apply it to PR-wide (e.g., Change all tables to Markdown). You might also want to run a quick spell check on the document!

File: docs/design/admonitions/index.mdx
Comment (dillp1) [reply docs/design/admonitions/index.mdx:21]: Yikes! I completely forgot I was working INSIDE the site I was linking to. Committing this suggestion.

File: docs/design/admonitions/index.mdx
Comment (dillp1) [reply docs/design/admonitions/index.mdx:77]: Good eye. Committing suggestion.

File: docs/design/admonitions/index.mdx
Comment (dillp1) [reply docs/design/admonitions/index.mdx:102]: I've been using HTML syntax for tables since I found it easier to edit in my IDE and for custom CSS class injections. I'll update it to markdown syntax for this repo.

Comment (dillp1) [reply docs/design/admonitions/index.mdx]: Resolved in https://github.com/DocusaurusCommunity/website/pull/32/commits/d6971301b03587f28afd339fb0ed4f5ab20a6b22

File: docs/design/admonitions/index.mdx
Comment (dillp1) [reply docs/design/admonitions/index.mdx:90]: I totally agree. Committing suggestion.

**Comment by dillp1** [comment]
I made some great changes and finally added that diagram (don't call me a graphic designer... I tried my best haha)

File: docs/design/admonitions/index.mdx
Comment (bohyunjung) [reply docs/design/admonitions/index.mdx]: Aha! Yup, if I had to inject an inline style specific to a table or if something could not be done with the Markdown syntax, I would also consider writing it down in HTML.

Thanks for the update!

**Review by bohyunjung — COMMENTED** [review]
Looks good to me! Thanks for applying the suggestions.

Before the merge, I noticed a mix of comma-separated and comma-less syntax for HTML rgb / rgba representations. Both work in modern browsers, but it'd be better if we stick to one style;

* I see [another in-project page](https://docusaurus.community/knowledge/design/css/variables/) has comma-separated expressions.
* However, surprisingly, according to the [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb) document, comma-less expression is standard, while comma-separated is legacy.

@dillp1 @homotechsual What's your opinion on which one to adopt?

**Comment by dillp1** [comment]
> Looks good to me! Thanks for applying the suggestions.
> 
> Before the merge, I noticed a mix of comma-separated and comma-less syntax for HTML rgb / rgba representations. Both work in modern browsers, but it'd be better if we stick to one style;
> 
> * I see [another in-project page](https://docusaurus.community/knowledge/design/css/variables/) has comma-separated expressions.
> * However, surprisingly, according to the [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb) document, comma-less expression is standard, while comma-separated is legacy.
> 
> @dillp1 @homotechsual What's your opinion on which one to adopt?

I've always used comma-separated expressions in my experience coding with Java, C, and Python. Ultimately, I think @homotechsual has a bit more experience than me and could give a more technical answer, but I would use comma-separated based on my experience.

**Comment by homotechsual** [comment]
> > Looks good to me! Thanks for applying the suggestions.
> > Before the merge, I noticed a mix of comma-separated and comma-less syntax for HTML rgb / rgba representations. Both work in modern browsers, but it'd be better if we stick to one style;
> > 
> > * I see [another in-project page](https://docusaurus.community/knowledge/design/css/variables/) has comma-separated expressions.
> > * However, surprisingly, according to the [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb) document, comma-less expression is standard, while comma-separated is legacy.
> > 
> > @dillp1 @homotechsual What's your opinion on which one to adopt?
> 
> I've always used comma-separated expressions in my experience coding with Java, C, and Python. Ultimately, I think @homotechsual has a bit more experience than me and could give a more technical answer, but I would use comma-separated based on my experience.

Definitely comma separated imo.

**Comment by homotechsual** [comment]
So basically if you can standardise the `rgb`/`rgba` definitions - either with our without commas just standardised - I think this is good to merge.

**Comment by bohyunjung** [comment]
@dillp1 Can you make another commit to making them all comma-separated? Let's get these merged!

**Comment by dillp1** [comment]
@bohyunjung will do when I’m free tomorrow! 😁

**Comment by dillp1** [comment]
Updated content to be more consisted with color values!

**Review by bohyunjung — APPROVED** [review]
👏 LGTM!

**Comment by bohyunjung** [comment]
@homotechsual Waiting for your approval and merge :)

**Review by homotechsual — APPROVED** [review]

**Pull request merged**
